EBA/Taschenrechner/MatheFunktionen/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
```

Comment:
maybe this won't work at school. this is new stuff!

Reply:
🤷‍♂️🤷‍♂️ lets solve this when i am back 2 work